src/docs/findNode.mdx

```diff
@@ -3,13 +3,33 @@
 # How to Find Your CKB Node ID
 <MarginContainer top={2} />
 
-<Tips text={"This guide will help you find your CKB Node ID when your ***CKB node or Neuron is up and running.*** You will need to open a new terminal window to interact with your node without disrupting its operation."} type={"normal"} />
+<Tips text={"This guide will help you find your CKB Node ID when your **CKB node or Neuron is up and running**."} type={"normal"} />
 
 <MarginContainer top={2.5} />
 
 ## Step 1: Retrieve your Node ID
 <MarginContainer top={0.5} />
 
+In the address bar of your browser, paste in the below text, and hit enter:
+
+#### Method 1:
+
+```html
+data:text/html
+<script>
+  fetch("http://127.0.0.1:8114", {method: "POST", headers:{"Content-Type": "application/json"},
+        body: '{"id":0,"jsonrpc":"2.0","method":"local_node_info","params":[]}'})
+  .then(response => response.json()).then(response => document.write(JSON.stringify(response)));
+</script>
+```
+<MarginContainer top={1.5} />
+
+#### Method 2:
+
+<GetRequestInfo />
```

Comment:
Method 1: 
Button

Method 2: 
In the address bar of your browser, paste in the below text, and hit enter:
...